### ISSUE TITLE
In 02. PyTorch Neural Network Classification 

### ISSUE THREAD

**Issue by longzhe-csgo** [mrdbourke/pytorch-deep-learning#1134]
The formula displayed in Section 3.1 of the article is garbled.

**Comment by mrdbourke** [comment]
Hey @longzhe-csgo , thank you for the heads up, I've managed to fix the larger equations but still a slight issue on the inline math displaying.

I'll keep working on a fix!

See: https://www.learnpytorch.io/02_pytorch_classification/#31-going-from-raw-model-outputs-to-predicted-labels-logits-prediction-probabilities-prediction-labels

<img width="888" alt="Screenshot 2025-01-07 at 10 29 28 AM" src="https://github.com/user-attachments/assets/85d6c3f2-cb80-4f1d-9e50-d0fe0d79b4eb" />

**Comment by 37Lime** [comment]
Greetings,

For those who wants to see the equations while they are getting fixed, use "View Source Code" (located at the top of the chapters in the book). There, the equations appear just fine.
